charts/dataingestion-service/values-test2.yaml

```diff
@@ -0,0 +1,85 @@
+replicaCount: 1
+
+env: "test2"
+
+image:
+  repository: 501715613725.dkr.ecr.us-east-1.amazonaws.com/cdc-nbs-modernization/data-ingestion-service
```

Comment:
This should not be in the repo as it contains our account information. It is supplied via argocd